### PR TITLE
Rn story 45

### PR DESCRIPTION
**Pull request by raymondknguyen**
## Description
Completes User Story #45

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor (non-breaking change which maintains existing functionality)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## RSpec Results
```
156 examples, 0 failures, 3 pending

(97.93%) covered.

test is at 97 because of the skipped test.
```

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,7 +1,11 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
-  helper_method :cart, :current_user, :current_admin?, :current_merchant?
+  helper_method :cart, 
+                :current_user, 
+                :current_admin?, 
+                :current_merchant?, 
+                :current_merchant_employee?
```
Comment (iEv0lv3) [inline app/controllers/application_controller.rb:8]: Nice helper method!

File: db/migrate/20200224053217_add_default_to_items.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddDefaultToItems < ActiveRecord::Migration[5.1]
+  def change
+    change_column :items, :image, :string, :default => "https://i.pinimg.com/originals/97/1b/8d/971b8dbcde2119776efdd26f0bbd4f47.jpg"
```
Comment (iEv0lv3) [inline db/migrate/20200224053217_add_default_to_items.rb:3]: May want to add a `to: new_default` `from: old_default` so that the change is reversible.

**Review by d-atkins — COMMENTED** [review]
error: not able to leave image blank

**Review by iEv0lv3 — COMMENTED** [review]
Issue Found:
-Visitor can enter URI directly and create items
  -Should be name spaced and inherit a base controller with conditional check for `current_merchant?'

**Review by iEv0lv3 — APPROVED** [review]
Looks good! Thanks for updating.

**Review by d-atkins — APPROVED** [review]
Both issues fixed

**Pull request merged**